opensrp-reveal/src/test/java/org/smartregister/reveal/presenter/FormFragmentPresenterTest.java

```diff
@@ -141,6 +145,25 @@ public void testOnFoundMembersCount() throws JSONException {
         assertEquals(1, jsonArgumentCaptor.getValue().getInt("sleep_outdoors"));
     }
 
+    @Test
+    public void testOnFoundMembersCountExceptionHandling() {
+        JSONObject mockedForm = Mockito.mock(JSONObject.class);
+        doAnswer(invocation -> {
+            throw new JSONException("Hey! I crash!");
+        }).when(mockedForm).toString();
+        presenter.onFetchedMembersCount(new Pair<>(3, 1), mockedForm);
+        verify(view).startForm(jsonArgumentCaptor.capture());
+        verify(view).hideProgressDialog();
+    }
+
+    @Test
+    public void testShowBasicForm() {
+        Whitebox.setInternalState(presenter, "jsonFormUtils", jsonFormUtils);
+        when(jsonFormUtils.getFormJSON(any(), anyString(), any(), any())).thenReturn(new JSONObject());
+        presenter.showBasicForm("form");
+        verify(view).startForm(jsonArgumentCaptor.capture());
```

Comment:
Could we have an assert statement to verify that the captured json is same as the one returned by `getFormJSON`